### DISCUSSION TITLE
Bot to track community discussions

### DISCUSSION THREAD

**Discussion by gbarbon** [mia-platform/community#9] [General]
Hi everyone,
it would be nice to have a bot that writes the link of every new thread in the Community in a dedicated chat (in our internal chat). 
In such a way it would be easier to follow new discussions. Is it feasible?

**Comment by edocrippaofficial** [comment]
There is a useful webhook, it can be integrated with a script that writes the name and the url of the discussion in a chat, I suppose
https://github.com/community/community/discussions/2943

**Comment by davidebianchi** [comment]
I added a workflow to send the notification to a google chat webhook.
